ccan/ccan/opt/helpers.c

```diff
@@ -35,9 +35,15 @@ char *opt_set_invbool(bool *b)
 
 char *opt_set_bool_arg(const char *arg, bool *b)
 {
-	if (!strcasecmp(arg, "yes") || !strcasecmp(arg, "true"))
+	if (!strcasecmp(arg, "yes") ||
+	    !strcasecmp(arg, "true") ||
+	    !strcasecmp(arg, "on") ||
```

Comment:
Hmm, this would have to go upstream into ccan, which is possible, but I actually thinking having so many synonyms was a mistake...